extensionsGallery-insider.json

```diff
@@ -3183,6 +3183,63 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "65",
+					"extensionName": "machine-learning",
+					"displayName": "Machine Learning",
+					"shortDescription": "Machine Learning for SQL Databases",
+					"publisher": {
+						"displayName": "Microsoft",
+						"publisherId": "Microsoft",
+						"publisherName": "Microsoft"
+					},
+					"versions": [
+						{
+							"version": "0.1.0",
+							"lastUpdated": "05/05/2020",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/machine-learning/machine-learning-0.1.0.vsix"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Icons.Default",
+									"source": "https://raw.githubusercontent.com/microsoft/azuredatastudio/master/extensions/machine-learning/images/extensionIcon.png"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/machine-learning/README.md"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Code.Manifest",
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/extensions/machine-learninge/package.json"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.License",
+									"source": "https://raw.githubusercontent.com/Microsoft/azuredatastudio/master/LICENSE.txt"
+								}
+							],
+							"properties": [
+								{
+									"key": "Microsoft.VisualStudio.Code.ExtensionDependencies",
+									"value": ""
+								},
+								{
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": ">=0.32.1"
```

Comment:
no Microsoft.AzDataEngine version dependency?

Reply:
I don't know why some extensions have that dependency some don't. @alanrenmsft do I need to add that dependency?

Reply:
IIRC, MS.VS.Code.Engine if required, azDataEngine was added by me to make it easy to control the ads version requirement since there is no 1 to 1 relation between ads version and the vscode version. for your case, if you have a dependency on the ADS core, you should add the Microsoft.VisualStudio.Code.azDataEngine key and set the value to be >=1.18.0

Reply:
ok updated